libs/sdk-backend-tiger/src/backend/workspace/catalog/datasetLoader.ts

```diff
@@ -121,22 +123,30 @@ function createDateDatasets(attributes: AttributeResourcesResponseSchema): ICata
     });
 }
 
+// TODO handle filtering by tags
+// "filter[tags.id]": includeTags
+// TODO - include "tags" is not possible
 export async function loadAttributesAndDateDatasets(
-    client: ITigerClient,
-    includeTags: string[],
+    sdk: ITigerClient,
+    workspaceId: string,
+    //    includeTags: string[],
 ): Promise<CatalogItem[]> {
-    const attributes = await client.metadata.attributesGet(
+    const attributesPromise = await sdk.workspaceModel.getEntities(
```

Comment:
Naming: since you `await` the variable no longer represents a promise, so the name is misleading. What about `attributesResponse`?